CONTRIBUTING.md

```diff
@@ -14,13 +14,16 @@ Add code owner per tool in [CODEOWNER](https://github.com/Azure/azure-sdk-tools/
 
 ## README
 
-Please add README.md file for every tool to illustrate:
+1. Please add README.md file for every tool to illustrate:
```

Comment:
The way these numbered items render is probably not what you intend:

![image](https://user-images.githubusercontent.com/1020379/145085617-0bc6b39b-85c7-4d7a-ab28-dcf75a440c79.png)